externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -152,6 +155,9 @@ class ExecuteStatement(
 
   private def runOperation(operation: Operation): Unit = {
     val result = executor.executeOperation(sessionId, operation)
+    if (result.getJobClient.isPresent) {
+      jobId = Some(result.getJobClient.get().getJobID)
+    }
```

Comment:
There is helper class `FlinkOperation#RichOptional` could simplify this code.

Reply:
I'm not sue if `FlinkOperation#RichOptional` helps, because it's not simply converting java `Optional<JobClient>` to scala `Option[JobClient]`,  but getting a `JobID` from `Optional<JobClient>` then making it a `Option[JobID]`. Please correct me if I'm wrong.